configure_ubuntu_16.sh

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env bash
 echo "Starting GDB ImageWatch configuration for Ubuntu 16.04."
 set -e
-sudo apt-get install python3-pip
-sudo apt-get -y install libpython3-dev libglew-dev python3-numpy python3-pip texinfo libfreetype6-dev libeigen3-dev
-sudo pip3 install pysigset
-mkdir build
+sudo apt update
+sudo apt install -y python3-pip
+sudo apt install -y libpython3-dev libglew-dev python3-numpy python3-pip texinfo libfreetype6-dev libeigen3-dev
+sudo -H pip3 install pysigset
+mkdir -p build
 cd build
 export QT_SELECT=5
 qmake ..
 make -j$(nproc)
-make install
+sudo make install
```

Comment:
Running the `make install` step as root is actually not desirable since in this script the installation is configured to be performed in the build folder itself, which does not require special permissions. If the user installs as a root user, this will probably make it harder to update the plugin in the future, since he will not have permissions to override the installed files (some of which are copied at build time, such as the scripts).

Could you please change the `sudo make install` to `make install`?

Reply:
Hi,
You are very welcome, great work on this project!

I understand.
However, if `sudo` is to be dropped, then line 11 would have to be extended with a `PREFIX=` statement, as this defaults to `usr/local` (see [gdb-imagewatch.pro:80](https://github.com/csantosbh/gdb-imagewatch/blob/master/gdb-imagewatch.pro#L80)).

The other alternative is to skip `make install` altogether, but I don't know if that would affect my IDE's ability to locate **gdb-imagewatch**? 

In the current version, it integrated perfectly with debugging with gdb in CLion. I don't know if the IDE finds the one in `usr/local/bin/gdb-imagewatch` or in the build folder. Any thoughts?

Reply:
Hello there! I'm sorry for the late reply.

You are right! Versions 1.1.x install to /usr/local by default. However, the configuration that is inserted to ~/.gdbinit by the installation script refers to the gdb-imagewatch.py located under the build folder rather than the one in the installation folder. Unfortunately, the whole deployment system was not well designed in the versions 1.1.x (for instance, the FONT_PATH preprocessor, which contains the path to the font file used for rendering the buffer values, also refers to the build folder rather than the installation folder).

I expect to have all these issues fixed by the time version 1.2 is released (I've also been working on removing as many dependencies as possible to simplify the installation process, so the configure_ubuntu_16.sh script will also look a lot simpler too).

For the time being, I think we can actually just drop the `make install` step for versions 1.1.x. What do you think?

Reply:
Hi,
I agree then that `make install` can be dropped. I'll make a new commit.
Do you want me to squash the previous commit?